package.json

```diff
@@ -14,6 +14,15 @@
     "test-exports": "lerna run test-exports",
     "test-unit": "yarn test-unit-esm && lerna run test-unit",
     "test-unit-esm": "mocha test/esm-named-exports.test.js",
+    "test-integration-mariadb": "yarn lerna run test-integration-mariadb",
+    "test-integration-mysql": "yarn lerna run test-integration-mysql",
+    "test-integration-postgres": "yarn lerna run test-integration-postgres",
+    "test-integration-postgres-native": "yarn lerna run test-integration-postgres",
+    "test-integration-sqlite": "yarn lerna run test-integration-sqlite",
+    "test-integration-mssql": "yarn lerna run test-integration-mssql",
+    "test-integration-db2": "yarn lerna run test-integration-db2",
+    "test-integration-ibmi": "yarn lerna run test-integration-ibmi",
+    "test-integration-snowflake": "yarn lerna run test-integration-snowflake",
```

Comment:
2 remarks;
1. Do we already have an use case for this? The only integration tests we have right now are in core, so it's just as easy to run this from inside that folder (or use --scope).
2. Postgres-native should not be the same as postgres.

Reply:
Building is done from the root folder, so if you want to test changes you're currently working on, you need to run `yarn build` in one terminal, and `yarn test-integration-x` in another, or use `yarn build && yarn lerna run test-integration-postgres`. This simplifies it to `yarn build && yarn test-integration-postgres`. With `yarn build` being cached, I'd like to simplify it further to just `yarn test-integration-postgres` by moving the build step inside of the script

Reply:
That sounds good. Once point 2 is fixed I'll approve this so we can get the first alpha out.

Reply:
Done!